File: Dataset/Leetcode/train/121/268.java
 class Solution {
    public int XXX(int[] prices) {
        int start = prices[0], end = prices[0], max = 0;
        for(int price : prices){
            start = Math.min(start, price);
            end = price;
            max = Math.max(max, end - start);
        }
        return max;
    }
}

File: Dataset/Leetcode/train/62/268.cpp
 class Solution {
public:
    int XXX(int m, int n) {
        int A[100] = { 0 };
        for (int i = 0; i < n; i++) A[i] = 1;
        for (int i = 1; i < m; i++) 
            for (int j = 1; j < n; j++)
                A[j] = A[j - 1] + A[j];

        return A[n - 1];
    }
};

